### ISSUE TITLE
Cluster Serving Can not create a Path from an empty string

### ISSUE THREAD

**Issue by qiyuangong** [intel-analytics/analytics-zoo#3807]
```
+ /opt/flink-1.10.1/bin/flink run -c com.intel.analytics.zoo.serving.ClusterServing -m 10.239.166.44:8081 /opt/analytics-zoo/analytics-zoo-bigdl_0.12.1-spark_2.4.3-0.10.0-SNAPSHOT-serving.jar
+ tee ./cluster-serving-job-sgx.log
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.apache.flink.api.java.ClosureCleaner (file:/opt/flink-1.10.1/lib/flink-dist_2.11-1.10.1.jar) to field java.lang.String.value
WARNING: Please consider reporting this to the maintainers of org.apache.flink.api.java.ClosureCleaner
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

------------------------------------------------------------
 The program finished with the following exception:

org.apache.flink.client.program.ProgramInvocationException: The main method caused an error: Can not create a Path from an empty string
        at org.apache.flink.client.program.PackagedProgram.callMainMethod(PackagedProgram.java:335)
        at org.apache.flink.client.program.PackagedProgram.invokeInteractiveModeForExecution(PackagedProgram.java:205)
        at org.apache.flink.client.ClientUtils.executeProgram(ClientUtils.java:138)
        at org.apache.flink.client.cli.CliFrontend.executeProgram(CliFrontend.java:662)
        at org.apache.flink.client.cli.CliFrontend.run(CliFrontend.java:210)
        at org.apache.flink.client.cli.CliFrontend.parseParameters(CliFrontend.java:893)
        at org.apache.flink.client.cli.CliFrontend.lambda$main$10(CliFrontend.java:966)
        at org.apache.flink.runtime.security.NoOpSecurityContext.runSecured(NoOpSecurityContext.java:30)
        at org.apache.flink.client.cli.CliFrontend.main(CliFrontend.java:966)
Caused by: java.lang.IllegalArgumentException: Can not create a Path from an empty string
        at org.apache.flink.core.fs.Path.checkAndTrimPathArg(Path.java:168)
        at org.apache.flink.core.fs.Path.<init>(Path.java:181)
        at org.apache.flink.optimizer.plantranslate.JobGraphGenerator.addUserArtifactEntries(JobGraphGenerator.java:286)
        at org.apache.flink.streaming.api.graph.StreamingJobGraphGenerator.createJobGraph(StreamingJobGraphGenerator.java:183)
        at org.apache.flink.streaming.api.graph.StreamingJobGraphGenerator.createJobGraph(StreamingJobGraphGenerator.java:104)
        at org.apache.flink.streaming.api.graph.StreamGraph.getJobGraph(StreamGraph.java:777)
        at org.apache.flink.streaming.api.graph.StreamGraphTranslator.translateToJobGraph(StreamGraphTranslator.java:52)
        at org.apache.flink.client.FlinkPipelineTranslationUtil.getJobGraph(FlinkPipelineTranslationUtil.java:43)
        at org.apache.flink.client.deployment.executors.ExecutorUtils.getJobGraph(ExecutorUtils.java:51)
        at org.apache.flink.client.deployment.executors.AbstractSessionClusterExecutor.execute(AbstractSessionClusterExecutor.java:57)
        at org.apache.flink.streaming.api.environment.StreamExecutionEnvironment.executeAsync(StreamExecutionEnvironment.java:1733)
        at org.apache.flink.streaming.api.environment.StreamContextEnvironment.executeAsync(StreamContextEnvironment.java:94)
        at org.apache.flink.streaming.api.environment.StreamExecutionEnvironment.executeAsync(StreamExecutionEnvironment.java:1706)
        at org.apache.flink.streaming.api.environment.StreamExecutionEnvironment.executeAsync(StreamExecutionEnvironment.java:1690)
        at org.apache.flink.streaming.api.scala.StreamExecutionEnvironment.executeAsync(StreamExecutionEnvironment.scala:713)
        at com.intel.analytics.zoo.serving.ClusterServing$.executeJob(ClusterServing.scala:66)
        at com.intel.analytics.zoo.serving.ClusterServing$.main(ClusterServing.scala:74)
        at com.intel.analytics.zoo.serving.ClusterServing.main(ClusterServing.scala)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.apache.flink.client.program.PackagedProgram.callMainMethod(PackagedProgram.java:321)
        ... 8 more

```

**Comment by Litchilitchy** [comment]
seems the uploaded jar path is incorrect, could you print the jar path in `ClusterServing.scala`?

**Comment by Litchilitchy** [comment]
Turns out you have update the master code but did not update the config, please also check config in `scripts/cluster-serving/config.yaml` when you pull master

**Comment by qiyuangong** [comment]
Fixed by #3866
